plugins/performance-lab/includes/site-health/bfcache-compatibility-headers/helper.php

```diff
@@ -66,41 +66,15 @@ function perflab_bfcache_compatibility_headers_check(): array {
 	}
 
 	foreach ( (array) $cache_control_headers as $cache_control_header ) {
-		$cache_control_header = strtolower( $cache_control_header );
-		$found_directives     = array();
-		foreach ( array( 'no-store', 'no-cache', 'max-age=0' ) as $directive ) {
-			if ( str_contains( $cache_control_header, $directive ) ) {
-				$found_directives[] = $directive;
-			}
-		}
-
-		if ( count( $found_directives ) > 0 ) {
+		if ( str_contains( strtolower( $cache_control_header ), 'no-store' ) ) {
 			$result['label']       = __( 'The Cache-Control page header is preventing fast back/forward navigations', 'performance-lab' );
 			$result['status']      = 'recommended';
 			$result['description'] = sprintf(
-				'<p>%s %s</p>',
+				'<p>%s</p>',
 				wp_kses(
-					sprintf(
-						/* translators: %s: problematic directive(s) */
-						_n(
-							'The <code>Cache-Control</code> response header for an unauthenticated request to the home page includes the following directive: %s.',
-							'The <code>Cache-Control</code> response header for an unauthenticated request to the home page includes the following directives: %s.',
-							count( $found_directives ),
-							'performance-lab'
-						),
-						implode(
-							', ',
-							array_map(
-								static function ( $header ) {
-									return "<code>$header</code>";
-								},
-								$found_directives
-							)
-						)
-					),
+					__( "The <code>Cache-Control</code> response header for an unauthenticated request to the home page includes the <code>no-store</code> directive. This can affect the performance of your site by preventing fast back/forward navigations (via the browser's bfcache).", 'performance-lab' ),
```

Comment:
Not a blocker, but for the code constructs using placeholders would be nice, since they shouldn't be translatable - plus it avoids the need for translators to deal with HTML in the string, which is always good if it can be avoided.
```suggestion
					sprintf(
						/* translators: 1: header name, 2: problematic directive */
						__( "The %1$s response header for an unauthenticated request to the home page includes the %2$s directive. This can affect the performance of your site by preventing fast back/forward navigations (via the browser's bfcache).", 'performance-lab' ),
						'<code>Cache-Control</code>',
						'<code>no-store</code>'
					),
```

Reply:
That works ffa9c2576 (with a couple tweaks)